_posts/2022-08-24-kubeflow-1.6-release.md

```diff
@@ -0,0 +1,146 @@
+- [The SDK reference](https://kubeflow-pipelines.readthedocs.io/en/master/)
+- [Breaking changes](https://docs.google.com/document/d/1nCUUVRXexXbQ0LDkGHsMIBDSu1WvJA9Upy1JzybNVMk/edit) including an SDK change from `kfp.v2` to `kfp` 
+
+<br />
+
+For those users who are still relying on the KFP v1 functionality, Kubeflow v1.6 and its KFP v2 component are fully
+ tested and supported with the mature features in the legacy Kubeflow Pipelines v1.8 SDK. This provides the same
+ functionality that thousands of KFP v1 users leverage in production today.   
+
+<br />
+
+In addition to the KFP v2 alpha, Kubeflow 1.6 includes feature enhancements and operational improvements for Katib
+ users. These include support for:
+
+- The population based training algorithm, URL
+- Enhanced validation checks for configurations, which will save time debugging parameter misconfiguration
+- Security fixes
+- MetricsUnavailable Status support, which will make debugging easier.
+
+<br />
+
+In Kubeflow v1.6, the Training Operator Working Group added these valuable enhancements:
+
+- Python SDK for PyTorch, MXNet, MPI, XGBoost  
+- The Clientset (Golang) is also generated for PyTorch, MXNet, MPI, XGBoost   
+- Gang scheduling support for MPI
+
+<br />
+
+Kubeflow v1.6 has several functional and operational improvements to Kubeflow Notebooks, Central Dashboard, webapps and
+ controllers. The following highlights the closed PRs that were considered medium-sized or larger. You can review
+ the full list of the closed PRs [here](https://github.com/kubeflow/kubeflow/pulls?page=1&q=is%3Apr+is%3Aclosed).
+
+- [Support for K8s 1.22](https://github.com/kubeflow/kubeflow/issues/6353)
+- [PodDefaults webhook picking up new certificates](https://github.com/kubeflow/kubeflow/issues/5560) 
+- Show objects from all names spaces in Central Dashboard, Jupyter, Tensorboard, Volumes Mgr
+- Updated the hosting container registry of images
+- [CVE fixes for Central Dashboard](https://github.com/kubeflow/kubeflow/pull/6519)  
+
+<br />
+
+The Manifest Working Group contributed several enhancements in 1.6. These enhancements including the testing of each
+ Kubeflow’s component manifest for:
+
+- [Compatibility with K8s 1.22](https://github.com/kubeflow/manifests/pull/2230)
+- [The manifests that can be applied](https://github.com/kubeflow/manifests/pull/2230)
+- [The Pods of components that can become ready](https://github.com/kubeflow/manifests/pull/2230)
+- [The basic objects that can be created](https://github.com/kubeflow/manifests/pull/2249)
+
+<br />
+
+For model serving, Kubeflow v1.6 incorporates the KServe v0.8.0 release, which includes these enhancements:
+
+- ClusterServingRuntime and ServingRuntime CRDs 
+- A new Model Spec is introduced to the InferenceService Predictor Spec as a new way to specify models.
+- Support for Knative 1.0+ 
+- gRPC for transformer to predictor network communication
+- An alternative backend ModelMesh with multi-namespace support
```

Comment:
Suggest having this list item be:

`Multi-namespace support for the ModelMesh alternative backend`

Reply:
Updated